controllers/reconcilers/grafana/plugins_reconciler.go

```diff
@@ -63,37 +64,10 @@ func (r *PluginsReconciler) Reconcile(ctx context.Context, cr *v1beta1.Grafana,
 			return v1beta1.OperatorStageResultFailed, err
 		}
```

Comment:
NIT: Since you're already updating this region, would you rename the dashboard variables to `pluginSource` or similar?
`GrafanaDatasources` can also specify plugins to be installed in a Grafana instance, and the `dashboard` key in the log could be wrong.

Reply:
Since the preview does not work for code that did not get edited, here's a link to the block:
https://github.com/grafana/grafana-operator/pull/2162/files/2accd6c9db33ca9b68f23bf14f69f6ded7540f69#diff-c3b1ea03db4d59f0779391df3bb3cb7f7263c1599c3c63eabcdbc36e30175b34L57-L64

Reply:
@Baarsgaard Yeah, that's a nice improvement as naming in `Reconcile` looks a bit messy.
I've just pushed a new commit that addresses that. Please, let me know if you agree with the new naming :)